src/Components/IconEditorTab/IconEditorTab.tsx

```diff
@@ -24,14 +24,47 @@ export const IconEditorTab: React.FC<IconEditorTabProps> = ({ widget }) => {
           <div className="scrivito_detail_label">
             <span>Preview</span>
           </div>
-          <div className="icon-editor-preview">
-            <i
-              className={`bs-icon ${currentIcon}`}
-              style={{
-                color: color,
-              }}
-            ></i>
+          <div className="row">
+            <div className="col-auto">
+              <div className="icon-editor-preview">
+                <i
+                  className={`bs-icon ${currentIcon}`}
+                  style={{
+                    color: color,
+                  }}
+                ></i>
+                <span className="button-del" title="Clear icon">
+                  －
+                </span>
+              </div>
```

Comment:
The minus in the red "clear" button seems to be misaligned (a little bit too high and not in the center):

![Screenshot 2024-09-12 at 14 34 10](https://github.com/user-attachments/assets/fdc78214-ded7-4c91-b9dd-8ff8eaeac2b8)

Reply:
FIXED in #[3e3b10a]